### PR TITLE
NWO Azure update

### PR DESCRIPTION
**Pull request by thedoubl3j**

### CODE REVIEW

**Comment by thedoubl3j** [comment]
ping @jctanner @gundalow

**Comment by jctanner** [comment]
`RuntimeError: lib/ansible/modules/cloud/azure/azure_rm_aks does not exist in the source`

File: scenarios/nwo/azure.yml [hunk 1]
```diff
@@ -24,3 +31,164 @@ azcollection:
   - cloud/azure/azure_rm_subnet.py
   - cloud/azure/azure_rm_virtualmachine.py
   - cloud/azure/azure_rm_virtualnetwork.py
+  - cloud/azure/azure_rm_aks
```
Comment (gundalow) [inline scenarios/nwo/azure.yml]: Missing `.py`

File: scenarios/nwo/community.yml [hunk 2]
```diff
@@ -721,92 +719,10 @@ general:
...
-  - cloud/azure/azure_rm_virtualmachine_info.py
-  - cloud/azure/azure_rm_virtualmachineextension_info.py
-  - cloud/azure/azure_rm_virtualmachineimage_info.py
-  - cloud/azure/azure_rm_virtualmachinescaleset_info.py
-  - cloud/azure/azure_rm_virtualmachinescalesetextension_info.py
-  - cloud/azure/azure_rm_virtualmachinescalesetinstance_info.py
-  - cloud/azure/azure_rm_virtualnetwork_info.py
-  - cloud/azure/azure_rm_virtualnetworkpeering_info.py
-  - cloud/azure/azure_rm_webapp_info.py
+  - cloud/centurylink/__init__.py
```
Comment (mattclay) [inline scenarios/nwo/community.yml]: This doesn't belong in the scenario.

**Pull request merged**